### PR TITLE
Move the 'elif' case for checking 'os.path.exists' higher

### PR DESCRIPTION
**Pull request by fbinliu**
##### SUMMARY

The change is required so that sysv init will be correctly identified and set as service manager.

This necessary because in WSL Ubuntu (20.04.4 LTS), the elif check using method 'is_systemd_managed_offline' will lead ansible to set 'systemd' as service manager, while the true service manager is SysV Init, as shown by running command (on my local machine running WSL Ubuntu 20.04.3 LTS) show as follows:
```
$ ps -p 1 -o comm=
init

```
while by running 'll /sbin/init' do returns this:
```
$ ll /sbin/init
/sbin/init -> /lib/systemd/systemd*
```
Currently, without making this change, the user of Ansible on Windows WSL Linux (i.e. Ubuntu) like me have to use another argument 'use' when using module 'service' to start or stop a service.

which mislead ansible to think the service manager is 'systemd' while it is actually SysV Init.

##### ISSUE TYPE

- Bugfix Pull Request
##### ADDITIONAL INFORMATION

Currently, running the next command on Windows WSL Ubuntu 20.04.3 LTS (Windows 11) get this:
```
$ ansible mylaptop -i hosts -m service -a "name=apache2 state=started" --ask-become-pass
localhost | FAILED! => {
    "ansible_facts": {
        "discovered_interpreter_python": "/usr/bin/python3"
    },
    "changed": false,
    "msg": "Service is in unknown state",
    "status": {}
}
```

The issue is also reported here by me on StackOverFlow: 
https://stackoverflow.com/questions/70200273/failed-to-start-apache2-using-ansible-module-service-on-localhost-in-wsl-ubuntu

```paste below

```

### CODE REVIEW

File: lib/ansible/module_utils/facts/system/service_mgr.py [hunk 1]
```diff
@@ -137,10 +137,10 @@ def collect(self, module=None, collected_facts=None):
                 service_mgr_name = 'upstart'
             elif os.path.exists('/sbin/openrc'):
                 service_mgr_name = 'openrc'
+            elif os.path.exists('/etc/init.d/'):
+                service_mgr_name = 'sysvinit
```
Comment (samdoran) [inline lib/ansible/module_utils/facts/system/service_mgr.py:141]:
```suggestion
                service_mgr_name = 'sysvinit'
```

**Comment by mkrizek** [comment]
Is it possible to add tests?

**Comment by mkrizek** [comment]
We recommend using the `use` parameter for the `service` module instead fixing corner cases like this one.

If you have further questions please stop by IRC or the mailing list:
* IRC: #ansible on irc.libera.chat
* mailing list: https://groups.google.com/forum/#!forum/ansible-project
